packages/adapter-node/package.json

```diff
@@ -33,21 +33,21 @@
 		"prepublishOnly": "pnpm build"
 	},
 	"devDependencies": {
-		"@polka/url": "^1.0.0-next.21",
 		"@sveltejs/kit": "workspace:^",
 		"@types/node": "^16.18.6",
 		"c8": "^7.12.0",
-		"polka": "^1.0.0-next.22",
 		"rimraf": "^5.0.0",
-		"sirv": "^2.0.2",
 		"typescript": "^4.9.4",
 		"uvu": "^0.5.6"
 	},
 	"dependencies": {
+		"@polka/url": "^1.0.0-next.21",
 		"@rollup/plugin-commonjs": "^24.0.0",
 		"@rollup/plugin-json": "^6.0.0",
 		"@rollup/plugin-node-resolve": "^15.0.1",
-		"rollup": "^3.7.0"
+		"polka": "^1.0.0-next.22",
+		"rollup": "^3.7.0",
```

Comment:
why does rollup appear in the dependencies?

Reply:
The Node adapter does another round of bundling as part of the adapt phase, to bundle all dev dependencies of the app, which didn't happen during Vite's build phase.

Reply:
Ah woops, I thought it was added, but I just misread the diff - sorry for the noise and thanks for the explanation.